packages/react-storage/src/components/StorageBrowser/views/Controls/Table.tsx

```diff
@@ -127,20 +128,35 @@ export type RenderRowItem<T> = (row: T, index: number) => JSX.Element;
 interface TableControlProps<T> {
   data: T[];
   columns: Column<T>[];
+  handleDroppedFiles?: (files: File[]) => void;
   renderHeaderItem: RenderHeaderItem<T>;
   renderRowItem: RenderRowItem<T>;
 }
 
 export function TableControl<U>({
   data,
   columns,
+  handleDroppedFiles,
   renderHeaderItem,
   renderRowItem,
 }: TableControlProps<U>): React.JSX.Element {
   const ariaLabel = 'Table';
 
+  const { dragState, ...dropHandlers } = useDropZone({
+    acceptedFileTypes: [],
```

Comment:
Is `acceptedFileTypes` a required parameter of `useDropZone`?

Reply:
No, it is not. I had left it included as a reminder in case I determined there were file restrictions we needed to impose. But given that's not the case, to my understanding, it can be removed

Reply:
Removed